NAMESPACE

```diff
@@ -250,6 +250,7 @@ exportMethods(rep)
 exportMethods(setdiff)
 exportMethods(show)
 exportMethods(union)
+import(rlang)
```

Comment:
Extra dependency for the sake of `abort`?

Reply:
Yeah, just because we’re likely to use it for other tools eventually too. rlang has no other dependencies so we generally consider it a “free” dependency for tidyverse packages.